templates/SSDS-1.html.tmpl

```diff
@@ -142,7 +142,7 @@
                 <p>
                     A Bitbucket Pull Request (PR) is where contributors and reviewers meet to discuss and assert the readiness of a contribution to a Bitbucket repository comprising code, tests, configuration, or a combination of these.</br>
                     <br/>
-                    A person who creates a PR gets attributed as its author and nominates at least one reviewer. Once mandatory changes requested by the reviewers have been incorporated by the contributors, reviewers can approve the PR. An approval permits that the approved contribution can be merged from a temporary source branch into a longer living target branch inside the repository. A single person must not assume the roles of contributor and reviewer on the same PR (no self-approvals).<br/>
+                    A person who creates a PR gets attributed as its author and nominates at least one reviewer. Once mandatory changes requested by the reviewers have been incorporated by the contributors, reviewers can approve the PR. An approval permits that the approved contribution can be merged from a temporary source branch into a longer living target branch inside the repository.{{#if data.sections.sec2s3.isGxpProject}} A single person must not assume the roles of contributor and reviewer on the same PR (no self-approvals).{{/if}}<br/>
```

Comment:
@valituguran it appears that `isGxpProject is a property of documentation section 2.3 (via `data.sections.sec2s3`). However, `isGxPProject` should be a global property. Therefore I suggest you use `data.isGxPProject` instead.

Reply:
@metmajer the changes have been done.